gnomad/utils/transcript_annotation.py

```diff
@@ -21,61 +21,59 @@
 logger = logging.getLogger("transcript_annotation_utils")
 logger.setLevel(logging.INFO)
 
-TISSUES_TO_EXCLUDE = {
-    "v7": [
-        "Bladder",
-        "Brain_Spinalcord_cervicalc_1",
-        "Brain_Substantianigra",
-        "Cells_EBV_transformedlymphocytes",
-        "Cells_Transformedfibroblasts",
-        "Cervix_Ectocervix",
-        "Cervix_Endocervix",
-        "FallopianTube",
-        "Kidney_Cortex",
-        "MinorSalivaryGland",
-        "Ovary",
-        "Prostate",
-        "Testis",
-        "Uterus",
-        "Vagina",
-    ],
-    "v10": [
-        "Bladder",
-        "Cells_EBV_transformedlymphocytes",
-        "Cells_Culturedfibroblasts",
-        "Cervix_Ectocervix",
-        "Cervix_Endocervix",
-        "Colon_Transverse_MixedCell",
-        "Colon_Transverse_Mucosa",
-        "Colon_Transverse_Muscularis",
-        "FallopianTube",
-        "Kidney_Cortex",
-        "Kidney_Medulla",
-        "Liver_Hepatocyte",
-        "Liver_MixedCell",
-        "Liver_PortalTract",
-        "Ovary",
-        "Pancreas_Acini",
-        "Pancreas_Islets",
-        "Pancreas_MixedCell",
-        "Prostate",
-        "SmallIntestine_TerminalIleum_LymphoidAggregate",
-        "SmallIntestine_TerminalIleum_MixedCell",
-        "Stomach_MixedCell",
-        "Stomach_Mucosa",
-        "Stomach_Muscularis",
-        "Testis",
-        "Uterus",
-        "Vagina",
-    ],
-}
-"""
-List of tissues to exclude from pext analyses and mean pext across tissues. Includes
-reproductive tissues, cell lines, and any tissue with less than 100 samples in the
-specified GTEx version. Tissues in v7 were excluded from gnomAD v2 pext calculations,
-and tissues in v10 were excluded from gnomAD v4 pext. Expression across these tissues
-is still displayed in the gnomAD browser.
-"""
+REPRODUCTIVE_TISSUES = [
+    "FallopianTube",
```

Comment:
```suggestion
    "Cervix_Ectocervix",
    "Cervix_Endocervix",
    "FallopianTube",
```
Beryl's paper also lists endocervix and ectocervix here